src/com/sap/piper/FileUtils.groovy

```diff
@@ -18,4 +18,10 @@ class FileUtils implements Serializable {
         def file = new File(dir)
         if (file.list().size() == 0) throw new AbortException("'${file.getAbsolutePath()}' is empty.")
     }
+
+    static validateFile(filePath) {
```

Comment:
Reusing the methods explained in the next comment, the executable is returned, so it is necessary to check the executable that it is a file.

Reply:
fileExists does not fit with the method name convention used in this class.

Reply:
I still thing that fileExists is clearer

Reply:
Yes, but it does not fit the naming convention, so you go to block this PR because of one name?

Reply:
How does this relate to the new config framework?

Reply:
No I do not request changes because of the naming here. I do not request changes on that level usually. I provide also the reason(s) when requesting changes. The reasons here are

> Reason for requesting changes. Merging of the configuration layers should be done similar to vars/executeMaven.
Reason for usage of 'general' section for the homes should be provided or these properties should be retrieved from the step specific configuration layer.

as outlined by another comment.

Reply:
Again: Reusing the methods refactored to Utils, the executable is returned, so it is necessary to check the executable that it is a file.